typescript/src/start_script.ts

```diff
@@ -71,6 +74,9 @@ if (grpcServiceConfig) {
     `--typescript_gapic_opt="grpc-service-config=${grpcServiceConfig}"`
   );
 }
+if (packageName) {
+  protocCommand.push(`--typescript_gapic_opt="package-name=${packageName}"`);
```

Comment:
Hmmm. Will it actually concatenate options into one string if two instances of `--typescript-gapic-opt` are passed?  I was under impression we need to join them manually here.

Reply:
just printed out the `this.request.parameter` for `CodeGeneratorRequest` and it gives:
```
"grpc-service-config=/Users/xiaozhenliu/gapic-generator-typescript/build/test/protos/google/cloud/texttospeech/v1/texttospeech_grpc_service_config.json","package-name=@google-cloud/text-to-speech"
```

Seems it has some methods to combine all `--typescript_gapic_opt` arguments together.

Reply:
when `CodeGenerateRequest.decode(inputBuffer)`, apart from fields `fileToGenerate`, `protoFile`, `compilerVersion` I think any other arguments get combined and go to `parameter`, which is one big string.

Reply:
Then I really think it's better to split this string by a comma and have key value pairs.